cli.sh

```diff
@@ -0,0 +1,217 @@
+#!/bin/bash
+# This bash script is a wrapper for the ghas-enablement tool to make it easier to run
+# It is meant to be run from the command line in the same folder as the ghas-enablement tool
+#
+
+# Define the colors
+GREEN=$(tput setaf 2)
+RED=$(tput setaf 1)
+CYAN=$(tput setaf 6)
+GREY=$(tput setaf 8)
+YELLOW=$(tput setaf 3)
+BLUE=$(tput setaf 4)
+RESET=$(tput sgr0)
+
+# Check if the .env file exists
+# If it doesn't exist, copy the .env.example file to .env
+if [ ! -f .env ]; then
+    cp .env.sample .env
```

Comment:
If it doesn't exist, shouldn't we copy it and then short-exit the script and let the user know they need to fill it out? I worry we are "doing something" without telling the user. So my thoughts are we echo out something that tell the user what we are doing and carry on. Or, tell the user, and exit the script asking them to update the values. Welcome your thoughts.

Reply:
Added it with [07b83b0](https://github.com/NickLiffen/ghas-enablement/pull/111/commits/07b83b0838d4bb967bb72e80ed316a6df8c5f55c). Let me know if this makes it more clear.